tsl/src/continuous_aggs/create.c

```diff
@@ -525,19 +523,13 @@ mattablecolumninfo_create_materialization_table(MatTableColumnInfo *matcolinfo,
 		mattablecolumninfo_add_mattable_index(matcolinfo, mat_ht);
 
 	/* Initialize the invalidation log for the cagg. Initially, everything is
-	 * invalid.
-	 * Integer time dimensions have now functions. These settings
+	 * invalid.  Integer time dimensions have now functions. These settings
```

Comment:
The stuff about the now function makes no sense anymore and should probably be removed

Reply:
Rewrote paragraph.